fatal: detected dubious ownership in repository at 'D:/esp/Espressif/frameworks/esp-idf-v5.0.2'
'D:/esp/Espressif/frameworks/esp-idf-v5.0.2/.git' is owned by:
	'S-1-5-32-544'
but the current user is:
	'S-1-5-21-3206365353-760775048-727138610-1001'
To add an exception for this directory, call:

	git config --global --add safe.directory D:/esp/Espressif/frameworks/esp-idf-v5.0.2
fatal: detected dubious ownership in repository at 'D:/esp/Espressif/frameworks/esp-idf-v5.0.2'
'D:/esp/Espressif/frameworks/esp-idf-v5.0.2/.git' is owned by:
	'S-1-5-32-544'
but the current user is:
	'S-1-5-21-3206365353-760775048-727138610-1001'
To add an exception for this directory, call:

	git config --global --add safe.directory D:/esp/Espressif/frameworks/esp-idf-v5.0.2
fatal: detected dubious ownership in repository at 'D:/esp/Espressif/frameworks/esp-idf-v5.0.2/components/openthread/openthread'
'D:/esp/Espressif/frameworks/esp-idf-v5.0.2/components/openthread/openthread/.git' is owned by:
	'S-1-5-32-544'
but the current user is:
	'S-1-5-21-3206365353-760775048-727138610-1001'
To add an exception for this directory, call:

	git config --global --add safe.directory D:/esp/Espressif/frameworks/esp-idf-v5.0.2/components/openthread/openthread
CMake Error at run_serial_tool.cmake:55 (message):
  
  D:/esp/esp_idf/python_env/idf5.0_py3.11_env/Scripts/python.exe;;D:/esp/Espressif/frameworks/esp-idf-v5.0.2/components/esptool_py/esptool/esptool.py;--chip;esp32
  failed


